Docs/Configuration.tex

```diff
@@ -1565,8 +1549,8 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   Reports security violation during attempts to write to \texttt{db}, \texttt{dbx},
   \texttt{PK}, and \texttt{KEK} variables from the operating system.
 
-  \emph{Note}: This quirk mainly attempts to avoid issues with NVRAM implementations
-  with problematic defragmentation, such as select Insyde or \texttt{MacPro5,1}.
+  \emph{Note}: This quirk attempts to mitigate problems on NVRAM implementations
```

Comment:
Prefer original "avoid issues with" to new "mitigate problems on" (can see repeated "with", but believe it is necessary). (Other surrounding changes look reasonable.)

(Also, original author seems to be using "select" instead of "certain", so sometimes easier changes might come from just making that swap; but no other change needed.)

Reply:
Yes! "Certain" is the word I needed to find for "Select". Changed that but leaving the rest as is.

Reply:
Glad that "certain" took your liking, too!

Would want to mention that, from the original comment, I am still not at all keen on "mitigate problems on", and prefer original "avoid issues with" despite repeated word "with" shortly afterwards.

To be more clear about why, I would think that "mitigate problems on" is actually not correct usage, and that it should be "mitigate problems with". Even with that change, I would, I think, _still_ find the original wording slightly preferable; and also then with no real need to change it; but with less remaining difference remaining between the two.

Hope that makes sense!

Reply:
Changed.

Reply:
🙏